docs/schemaform/form-config.md

```diff
@@ -51,6 +52,11 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
       // JSON schema object for the page. Follows the JSON Schema format.
       schema: {
         type: 'object',
+        // In a schema's properties, there are references to definitions. For example:
+        //   "homePhone": { "$ref": "#/definitions/phone" }
+        // In the config file, the definition for phone would then need to be added into definitions
+        // for it to be parsed correctly and added to homePhone.
+        definitions: {},
```

Comment:
Thanks for adding to this document, though actually this still doesn't quite make sense to me. It looks like our vets-json-schema schema files typically have a definitions field (some subset of common definitions), and the corresponding form config file will have a defaultDefinitions field, which is yet another subset.

Guess I'm saying maybe we should make this example correspond more accurately to the way we're organizing our code.